framework/src/utils/Units.C

```diff
@@ -193,7 +193,8 @@ const std::vector<std::pair<std::string, MooseUnits>> MooseUnits::_unit_table =
        {MooseUnits::BaseUnit::SECOND, -2}}}},                        // bar (unit of pressure)
     {"h", {60 * 60, 0, {{MooseUnits::BaseUnit::SECOND, 1}}}},        // hour
     {"day", {60 * 60 * 24, 0, {{MooseUnits::BaseUnit::SECOND, 1}}}}, // day
-    {"l", {1e-3, 0, {{MooseUnits::BaseUnit::METER, 3}}}},            // liter
+    {"a", {60 * 60 * 24 * 365.2422, 0, {{MooseUnits::BaseUnit::SECOND, 1}}}}, // year (a for annum)
```

Comment:
No. `a` unit is 365.25 days.

Reply:
probably should have checked wikipedia first